src/features/defi/providers/univ2/hooks/useUniV2LiquidityPool.ts

```diff
@@ -186,40 +288,108 @@ export const useUniV2LiquidityPool = ({
         }
         return broadcastTXID
       } catch (error) {
-        moduleLogger.warn(error, 'useFoxEthLiquidityPool:addLiquidity error')
+        moduleLogger.warn(error, 'useUniV2LiquidityPool:addLiquidity error')
       }
     },
     [
       accountId,
       accountNumber,
       adapter,
-      asset0.chainId,
-      asset0.precision,
-      asset1.precision,
-      asset1ContractAddress,
+      asset0,
+      assetId0OrWeth,
+      assetId1OrWeth,
+      makeAddLiquidtyData,
       skip,
       supportedEvmChainIds,
       uniswapRouterContract,
       wallet,
+      weth,
     ],
   )
 
+  const makeRemoveLiquidityData = useCallback(
+    ({
+      asset0ContractAddress,
+      asset1ContractAddress,
+      lpAmount,
+      asset1Amount,
+      asset0Amount,
+    }: {
+      asset0ContractAddress: string
+      asset1ContractAddress: string
+      lpAmount: string
+      asset1Amount: string
+      asset0Amount: string
+    }) => {
+      if (!uniswapRouterContract) throw new Error('uniswapRouterContract not defined')
+
+      const lpAmountBaseUnit = bnOrZero(lpAmount)
+        .times(bn(10).exponentiatedBy(lpAsset.precision))
+        .toFixed(0)
+      const deadline = Date.now() + 1200000
```

Comment:
what is this value?

Reply:
20 mins

Reply:
https://github.com/shapeshift/web/pull/4152/commits/776f174056ce95fbb61e2851d14c90b4accc9f14